.github/workflows/ci.yml

```diff
@@ -29,8 +29,6 @@ jobs:
       - run: npm ci
       - name: Build Types
         run: npm run build:types
-      - name: Test Types
-        run: npm run test:types
       - name: Lint Types
```

Comment:
There is a `lint:types` step here which we don't seem to have in parse-server; what is that for if the types are auto-generated anyway? should we remove it from here, or add it to parse-server as well?

Reply:
This repo uses typescript the server uses flow type

Reply:
Can/should that be unified?

Reply:
They could be merged however it’s much harder to debug

Reply:
Let's leave as is